src/stcal/jump/jump.py

```diff
@@ -88,7 +92,21 @@ def detect_jumps(frames_per_group, data, gdq, pdq, err,
         A dictionary with at least the following keywords:
         DO_NOT_USE, SATURATED, JUMP_DET, NO_GAIN_VALUE, GOOD
 
+    after_jump_flag_dn1 : float
```

Comment:
could you add a slightly more descriptive docstring for these, it's not clear exactly what they control just based on this description. also, what do you think about grouping them into 2 tuples so there are not 4 new parameters (each n grouped with the corresponding dn?)

Reply:
I thought about tuples.  But thought that might be more complex to explain than explicit single values like all the other ones.

Reply:
Updated the docstring to (hopefully) be more understandable.